hardware/fpga/run.tcl

```diff
@@ -17,6 +17,9 @@ if {$::env(MAIN_MEM)=="HYPER"} {
     add_files -fileset constrs_1 -norecurse "alsaqr/tcl/fmc_board_hyper_$::env(BOARD).xdc"
 } elseif {$::env(MAIN_MEM)=="DDR4"} {
     add_files -fileset constrs_1 -norecurse "alsaqr/tcl/ddr_$::env(BOARD).xdc"
+    if {$::env(NUM_CORES)==4} {
+        set SRC_CLK_PERIOD 50
```

Comment:
Could you please double-check wrt this file?hardware/fpga/alsaqr/tcl/ips/clk_mngr/run.tcl

Could you please remind me what this config does in this file?